docs/diagrams/DeleteXYZTagActivityDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User enters DeleteXYZTagCommand with a specified index and parameter;
```

Comment:
if we define XYZ to be 'Specialisation, Medical Condition or Prescriptions', then there should be no 'Tag'